### ISSUE TITLE
Errors report "success" with exit code `0`.

### ISSUE THREAD

**Issue by rduplain** [borkdude/nbb#73]
**version**

```
$ which node; node --version
/usr/bin/node
v13.14.0
$ npm list -g | grep nbb
├── nbb@0.0.91
```

**platform**

Ubuntu 20.04 LTS with packaged Node.js v13.14.0.

**problem**

Scripting errors exit with status code `0` meaning "success."

**repro**

```
$ nbb -e '(print (* 6 7))'; echo "exit code: $?"
42
exit code: 0
$ nbb -e '(printt (* 6 7))'; echo "exit code: $?"
----- Error --------------------------------------
Message:  Could not resolve symbol: printt
Location: 1:2
Phase:    analysis
exit code: 0
$ 
```

**expected behavior**

Return anything but `0` as the exit code.

When errors return non-zero status code, `nbb` could be used as part of a larger script that checks its exit code.

**Comment by borkdude** [comment]
Should be fixed in v0.0.93 which is currently being built/deployed on CI.

**Comment by borkdude** [comment]
Wishful thinking, in CI tests failed.

**Comment by borkdude** [comment]
Ah, in Node 14 (which is used on CI) unhandled promises don't result in a non-zero exit code.

**Comment by borkdude** [comment]
Should be fixed in v0.0.95.

**Comment by rduplain** [comment]
Thanks ~ looks good!
